src/server/wtr-config.js

```diff
@@ -0,0 +1,157 @@
+import { argv } from 'node:process';
+import { defaultReporter } from '@web/test-runner';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+
+const DEFAULT_PATTERN = type => `./test/**/*.${type}.js`;
+const DEFAULT_VDIFF = false;
+const ALLOWED_BROWSERS = ['chromium', 'firefox', 'webkit'];
+
+export class WTRConfig {
+
+	static #singleton;
+	#cliArgs = [];
+
+	constructor() {
+		if (!this.constructor.#singleton) {
+			this.cliArgs = argv;
+			this.constructor.#singleton = this;
+		}
```

Comment:
I'm not totally sure what singleton stuff is doing... is it letting you default `cliArgs` to `argv`? Why not just take the CLI args as a parameter to the constructor?

Reply:
I started writing up an explanation of why I switched away from `constructor(cliArgs)` and realized it didn't work because I'm running my tests against the public `getBrowsers` instead of `WTRConfig.getBrowsers`. Fixing that now.

The singleton part is still useful to avoid unnecessarily creating new instances on every `getBrowsers` call. Not a huge hit to lose it, though.